cmd/main.go

```diff
@@ -0,0 +1,69 @@
+package cmd
```

Comment:
[Question]: I'm not sure you need a `cmd` package since the sniffer binary will never run as a CLI tool.
@matthyx What do you think about this?